LoggingKata/Program.cs

```diff
@@ -16,14 +16,35 @@ static void Main(string[] args)
 
             var lines = File.ReadAllLines(csvPath);
 
+            logger.LogInfo("Begin parsing");
+
             logger.LogInfo($"Lines: {lines[0]}");
 
             var parser = new TacoParser();
 
             var locations = lines.Select(parser.Parse).ToArray();
 
-            // TODO:  Find the two Taco Bells that are the furthest from one another.
-            // HINT:  You'll need two nested forloops
+            ITrackable farthestI = null;
+            ITrackable farthestJ = null;
+
+            double distance = 0;
+
+            for (int i = 0; i < locations.Length; i++)
+            {
+                for (int j = 0; j < locations.Length; j++)
+                {
+                    GeoCoordinate origin = new GeoCoordinate(locations[i].Location.Latitude, locations[i].Location.Longitude);
```

Comment:
This doesn't need to be re-calculated every time the "j" loop executes. Consider moving it above line 34 so that it's only created once per "i"